.github/workflows/release_drafter.yml

```
@@ -0,0 +1,23 @@
+name: Release Drafter
+
+permissions:
+  contents: read
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  updateReleaseDraft:
+    permissions:
+      contents: write
+      pull-requests: read
+    runs-on: ubuntu-22.04
```

```suggestion
    runs-on: ubuntu-latest
```

Is it necessary to pin to a specific, older Linux version? If not, could we just use `ubuntu-latest` so this doesn't need to be updated whenever GitHub drops Ubuntu 22.04 runners?

---

It's a copy-paste artifact from `workflows/no-response.yml`.

I guess it was there since #5288, so now we can safely use `latest`